src/instructlab/eval/exceptions.py

```diff
@@ -90,3 +90,15 @@ def __init__(self, tasks_dir) -> None:
         super().__init__()
         self.tasks_dir = tasks_dir
         self.message = f"Invalid Tasks Dir: {tasks_dir}"
+
+
+class OpenAIError(EvalError):
```

Comment:
Since the underlying API could change, maybe this should be more generic. ModelServingAPIError?